### PR TITLE
 added helm test

### PR DESCRIPTION
**Pull request by himanshu2464**

### CODE REVIEW

File: flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/admin-ui/values.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ resources:
     memory: 2500Mi
 service:
   # -- The name of the admin ui port within the admin service. Please keep it as default.
-  name: http-admin-ui
```
Comment (moabu) [inline flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/admin-ui/values.yaml:49]: do not change these

File: flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/auth-server/values.yaml [hunk 2]
```diff
@@ -47,7 +47,7 @@ resources:
     memory: 2500Mi
 service:
   # -- The name of the oxauth port within the oxauth service. Please keep it as default.
-  name: http-auth
```
Comment (moabu) [inline flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/auth-server/values.yaml:50]: do not change these

File: flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/config-api/values.yaml [hunk 3]
```diff
@@ -52,7 +52,7 @@ resources:
     memory: 2500Mi
 service:
   # -- The name of the config-api port within the config-api service. Please keep it as default.
-  name: http-config-api
```
Comment (moabu) [inline flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/config-api/values.yaml:55]: do not change these

**Review by moabu — CHANGES_REQUESTED** [review]
Although these work the response back from them aren't entirely accurate. Some of these endpoints return a dictionary object showing the database status as well. Think of a way to call the python script listed in to execute the parse of these. Also think about adding a test connection for the database as well.

File: flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/auth-server/values.yaml (hunk 2 above)
Comment (himanshu2464) [reply flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/auth-server/values.yaml:50]: okay

File: flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/config-api/values.yaml (hunk 3 above)
Comment (himanshu2464) [reply flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/config-api/values.yaml:55]: okay

**Comment by himanshu2464** [comment]
> Although these work the response back from them aren't entirely accurate. Some of these endpoints return a dictionary object showing the database status as well.
> 
> * [ ]  Think of a way to call the python script listed in to execute the parse of these.
> * [ ]  Think about adding a test connection for the database as well. Hint: its already included in one of the services.
> * [ ]   Add a workflow that executes testing it

okay, we can add the script to check the database connection and also a python or bash script.

**Comment by moabu** [comment]
> > Although these work the response back from them aren't entirely accurate. Some of these endpoints return a dictionary object showing the database status as well.
> > 
> > * [ ]   Think of a way to call the python script listed in to execute the parse of these.
> > * [ ]   Think about adding a test connection for the database as well. Hint: its already included in one of the services.
> > * [ ]    Add a workflow that executes testing it
> 
> okay, we can add the script to check the database connection and also a python or bash script. and ya sure I will add the workflow to execute testing it.

Think in terms of what already exists in the containers  https://github.com/GluuFederation/flex/blob/main/flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/values.yaml#L187

**Pull request closed without merging**